network/p2pNetwork.go

```diff
@@ -754,29 +783,108 @@ func (n *P2PNetwork) OnNetworkAdvance() {
 	}
 }
 
+// TelemetryGUID returns the telemetry GUID of this node.
+func (n *P2PNetwork) TelemetryGUID() string {
+	return n.log.GetTelemetryGUID()
+}
+
+// InstanceName returns the instance name of this node.
+func (n *P2PNetwork) InstanceName() string {
+	return n.log.GetInstanceName()
+}
+
+// GenesisID returns the genesis ID of this node.
+func (n *P2PNetwork) GenesisID() string {
+	return n.genesisID
+}
+
+// NetProtoVersions returns the supported protocol versions of this node.
```

Comment:
I'd consider keeping the naming consistent 'SupportdeProtocolVersions'.

Reply:
done